sympy/series/formal.py

```diff
@@ -1219,12 +1219,13 @@ def coeff_bell(self, n):
         The second kind of Bell polynomials (are sometimes called "partial" Bell
         polynomials or incomplete Bell polynomials) are defined as
 
-        .. math:: B_{n,k}(x_1, x_2,\dotsc x_{n-k+1}) =
+        .. math::
+            B_{n,k}(x_1, x_2,\dotsc x_{n-k+1}) =
                 \sum_{j_1+j_2+j_2+\dotsb=k \atop j_1+2j_2+3j_2+\dotsb=n}
                     \frac{n!}{j_1!j_2!\dotsb j_{n-k+1}!}
-                    \left(\frac{x_1}{1!} \right)^{j_1}
-                    \left(\frac{x_2}{2!} \right)^{j_2} \dotsb
-                    \left(\frac{x_{n-k+1}}{(n-k+1)!} \right) ^{j_{n-k+1}}.
+                        \left(\frac{x_1}{1!} \right)^{j_1}
+                            \left(\frac{x_2}{2!} \right)^{j_2} \dotsb
+                                \left(\frac{x_{n-k+1}}{(n-k+1)!} \right) ^{j_{n-k+1}}.
```

Comment:
Why do you need to indent like staircase?

Reply:
I thought, it is a common style.

Reply:
In my opinion, since [PEP8](https://www.python.org/dev/peps/pep-0008/#id19) recommendation have 72 character lines, I'd rather not indented any LaTeX formula unless  necessary because it forces the lines to be longer.

Reply:
You are right, LaTeX formula is already huge and indentation makes it worse. Thanks for the input... Tried to resolve it, please review the commit.